src/project/internal/projectfilescontroller.cpp

```diff
@@ -486,7 +487,13 @@ io::path ProjectFilesController::selectScoreOpeningFile()
            << QObject::tr("MuseScore Dev Files") + " (*.mscs)"
            << QObject::tr("MuseScore Backup Files") + " (*.mscz~)";
 
-    return interactive()->selectOpeningFile(qtrc("project", "Score"), "", filter.join(";;"));
+    return interactive()->selectOpeningFile(qtrc("project", "Score"), ProjectFilesController::defaultOpenFileDir() , filter.join(";;"));
+}
+
+io::path ProjectFilesController::defaultOpenFileDir() const {
+    QString openFilePath = QDir::cleanPath(QDir::homePath() + QDir::separator() + (QString)"Documents/MuseScore/Scores");
+    io::path pathToDefaultOpenFileDir(openFilePath);
+    return pathToDefaultOpenFileDir;
```

Comment:
Hi! Thanks for your contribution.

I think it is better to use the value from `projectConfiguration()->userProjectsPath()` instead. The user can customize this value in Preferences, and `projectConfiguration()->userProjectsPath()` retrieves the current path, in the correct format.